scripts/index-demo-page.js

```diff
@@ -446,6 +482,15 @@
       var ext;
       var type = stateEls.type.value;
 
+      // reset type if it's a manifest object's type
+      if (type === 'application/vnd.videojs.vhs+json') {
```

Comment:
always reset if the type is 'application/vnd.videojs.vhs+json' as we don't really want to use it between sources.